pkg/services/cloudmigration/cloudmigrationimpl/gcomstub.go

```diff
@@ -58,3 +58,7 @@ func (client *gcomStub) CreateToken(ctx context.Context, params gcom.CreateToken
 	}
 	return token, nil
 }
+
+func (client *gcomStub) DeleteToken(ctx context.Context, params gcom.DeleteTokenParams) error {
+	return nil
```

Comment:
After merging https://github.com/grafana/grafana/pull/88176, update this to set `client.token` to nil.

Reply:
Done